README.md

```diff
@@ -1 +1,2 @@
 Readme
+oh, boh boh boh boh.
```

Comment:
This is me leaving a comment. Comments are useful when you want to make suggestions or ask questions about somebody's changes to a file. 

Using back-ticks, you can leave code in the comments:
```
The Readme is important to explain the purpose of your github repo
```

There are also `inline` code comments. 

Once somebody approves of the pull request you can click the "merge" button and it will write over the `master` branch (the branch we started with).

Great work!